project/Dependencies.scala

```diff
@@ -52,6 +52,8 @@ object Dependencies {
     "org.broadinstitute.dsde.workbench" %% "workbench-oauth2" % s"0.2-$workbenchLibsHash",
     "org.broadinstitute.dsde.workbench" %% "sam-client"       % "0.1-ef83073",
     "org.broadinstitute.dsde.workbench" %% "workbench-notifications" %s"0.3-$workbenchLibsHash",
+    // workbench-google2 has jose4j as a dependency; directly updating to a non-vulnerable version until workbench-google2 updates
+    "org.bitbucket.b_c" % "jose4j" % "0.9.3",
```

Comment:
I think this should move to the `transitiveDependencyOverrides` block at line 21, to prevent including `jose4j` unintentionally if anything about `workbench-google2` changes

Reply:
Ah ok thank you